app/static/init.uv.js

```diff
@@ -0,0 +1,86 @@
+const script = document.getElementById("init-uv");
+const manifest_url = script.getAttribute("manifest_url");
+
+function initUniversalViewer() {
+  const data = {
+    manifest: manifest_url,
+    embedded: true,
+  };
+
+  const uv = UV.init("uv", data);
+  uv.on("configure", function ({ config, cb }) {
+    config.modules.centerPanel.options.usePdfJs = true;
+    config.modules.footerPanel.options = {
+      downloadEnabled: false,
+      embedEnabled: false,
+      fullscreenEnabled: false,
+      moreInfoEnabled: false,
+      shareEnabled: false,
+    };
+    cb({
+      options: {
+        footerPanelEnabled: false,
+        leftPanelEnabled: true,
+        rightPanelEnabled: false,
+        headerPanelEnabled: true,
+      },
+      pdfHeaderPanel: {
+        options: {
+          centerOptionsEnabled: false,
+        },
+      },
+    });
+  });
+}
+
+document.addEventListener("DOMContentLoaded", function () {
+  initUniversalViewer();
+});
+
+document.querySelectorAll(".govuk-tabs__tab").forEach((tab) => {
```

Comment:
could we decouple the tab switching logic from the javascript and instead? If we do that, we should then be able to reuse the tabs for when javascript is disabled also like we discussed in the demo

Reply:
Its been a while since the call so yeah maybe wortth catching up again but my thoughts are, if we support the tab switching with non javascript then we'd need to do it via form submissions and thus wouldnt need javascript to do it.

Reply:
Check with Terry that UI side of changes is ok